src/app/pages/quickMaps/components/download/download.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="container-row">
+  <div class="element-container">
+    <i class="fas fa-file-image"></i>
+    <a [href]="chartDownloadPNG" target="_blank">
```

Comment:
Please could you add the `referrerpolicy="no-referrer-when-downgrade"` attribute?  This will allow the quickchart service to detect the URL query string of the requesting page so the QR code can be updated to point directly to the page the graphic came from?

Reply:
added